ush/setup.py

```diff
@@ -523,23 +560,27 @@ def _remove_tag(tasks, tag):
     # for all observation types.
     if not vx_fields_config:
         metatask = "metatask_check_post_output_all_mems"
-        rocoto_config['tasks'].pop(metatask)
+        rocoto_config["tasks"].pop(metatask)
 
     # If for a given obstype no fields are specified, remove all vx metatasks
     # for that obstype.
     for obstype in vx_fields_all:
-        vx_fields_obstype = [field for field in vx_fields_config if field in vx_fields_all[obstype]]
+        vx_fields_obstype = [
+            field for field in vx_fields_config if field in vx_fields_all[obstype]
+        ]
         if not vx_fields_obstype:
             for metatask in vx_metatasks_all[obstype]:
-                if metatask in rocoto_config['tasks']:
-                    logging.info(dedent(
-                        f"""
+                if metatask in rocoto_config["tasks"]:
+                    logging.info(
+                        dedent(
```

Comment:
I really like this argument. I think it will be out of scope for the set up main SRW integration PRs, but I'd happily come back and take out this nonsense in a cleanup  PR!

Some SRW developers really want to control how messages show up in the console, too.